calico/tests/conftest.py

```diff
@@ -23,10 +23,14 @@
 
 def setup_calico():
```

Comment:
This could be done in a separate PR, but it would be nice to have the calico version set as a variable and configuration in `tox.ini`